### DISCUSSION TITLE
about new created Flicker 30k grounding annotation

### DISCUSSION THREAD

**Discussion by peiliu0408** [Vision-CAIR/MiniGPT-4#379] [Q&A] [answered]
There is no doubt that this is an amazing work, gpt4-v2, particularly due to its efficient training. I am curious to know if there are any plans to make the Flicker 30K dataset available?

**Comment by junchen14** [comment, marked as answer]
we will make the filtered Flicker 30k dataset available in few days. Stay tuned!
